### PR TITLE
Specify the CLR

### PR DESCRIPTION
**Pull request by wuyuansushen**
## Summary

Provide more details

### CODE REVIEW

File: docs/csharp/language-reference/operators/lambda-expressions.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ A lambda expression with an expression on the right side of the `=>` operator is
 (input-parameters) => expression
 ```
 
-The body of an expression lambda can consist of a method call. However, if you are creating [expression trees](../../programming-guide/concepts/expression-trees/index.md) that are evaluated outside the context of the .NET common language runtime, such as in SQL Server, you should not use method calls in lambda expressions. The methods will have no meaning outside the context of the .NET common language runtime.
+The body of an expression lambda can consist of a method call. However, if you are creating [expression trees](../../programming-guide/concepts/expression-trees/index.md) that are evaluated outside the context of the .NET Common Language Runtime(CLR), such as in SQL Server, you should not use method calls in lambda expressions. The methods will have no meaning outside the context of the .NET Common Language Runtime(CLR).
```
Comment (pkulikov) [inline docs/csharp/language-reference/operators/lambda-expressions.md:53]: White spaces are missing before opening parentheses

Comment (BillWagner) [reply docs/csharp/language-reference/operators/lambda-expressions.md:53]: Here's the fix:

```suggestion
The body of an expression lambda can consist of a method call. However, if you are creating [expression trees](../../programming-guide/concepts/expression-trees/index.md) that are evaluated outside the context of the .NET Common Language Runtime (CLR), such as in SQL Server, you should not use method calls in lambda expressions. The methods will have no meaning outside the context of the .NET Common Language Runtime (CLR).
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks for spotting this and making the fix @wuyuansushen 

As @pkulikov noted, the space before the opening parentheses are missing. If you accept the suggestion I made, this is ready to merge.

Thanks again.

**Comment by wuyuansushen** [comment]
@BillWagner Sure, Please merge this fork, Thanks!

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks again @wuyuansushen 
This was ready to :shipit:, and I've merged it. You should see the changes live tomorrow on our regular publishing cycle.

**Pull request merged**